plugins/modules/vyos_firewall_global.py

```diff
@@ -359,6 +374,7 @@
 #        "set firewall config-trap 'enable'",
 #        "set firewall state-policy established action 'accept'",
 #        "set firewall state-policy established log 'enable'",
+#        "set firewall state-policy established log-level 'emerg'",
```

Comment:
Incorrect path (1.3.8)
```
vyos@client-11# set firewall state-policy established log-level

  Configuration path: firewall state-policy established [log-level] is not valid
  Set failed


```

Reply:
Good catch, this module was updated, docs were not.